x-pack/plugins/security_solution/public/flyout/document_details/isolate_host/header.tsx

```diff
@@ -33,6 +34,12 @@ export const PanelHeader: FC = () => {
     <EuiFlexGroup responsive gutterSize="s">
       <EuiFlexItem grow={false} data-test-subj="flyoutHostIsolationHeaderTitle">
         {isolateAction === 'isolateHost' ? <>{ISOLATE_HOST}</> : <>{UNISOLATE_HOST}</>}
+        <EuiSpacer size="s" />
+        <AgentTypeIntegration
+          agentType={agentType}
+          layout="horizontal"
```

Comment:
I personally think the vertical layout looks best everywhere. Alternatively, I think just using the `?` icon to the right of the logo should be enough and no need to use `Integration` along with it. 🤔 

![Screenshot 2024-06-27 at 14 50 23](https://github.com/elastic/kibana/assets/1849116/916b1a85-1416-4094-99e7-34f0f806131c)

Reply:
For me, the vertical layout did not display correctly in the Flyout header (from alerts). I highlighted this to @elasticusnick on the team's working issue. 

Can you add these comments there so that we can discuss with him? its issue 9412